package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "slackbot",
+  "version": "0.0.0",
+  "description": "Slack Bot that displays your current bonus balance",
+  "main": "src/app.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/app.js"
+  },
+  "author": "",
+  "dependencies": {
+    "@slack/bolt": "^4.0.1",
+    "@slack/rtm-api": "^7.0.1",
+    "@slack/web-api": "^7.7.0",
+    "body-parser": "^1.20.3",
+    "dotenv": "^16.4.5"
```

Comment:
Move `dotenv` to `devDependencies` in the `package.json`.

It is a good practice to separate the app's dependencies from those used solely for its development. It is then easier to see which packages are deployed with the application code and which are just local stuff.